src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt

```diff
@@ -186,11 +186,13 @@ Referenced assemblies:
 - 'Microsoft.VisualStudio.Shell.Immutable.14.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
 - 'Microsoft.VisualStudio.Shell.Interop, Version=7.1.40304.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
 - 'Microsoft.VisualStudio.Shell.Interop.8.0, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
+- 'Microsoft.VisualStudio.Utilities, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
```

Comment:
Due to DelegateCommand usage.
I'm a bit surprised with this change -- it was already used in the project: https://github.com/SonarSource/sonarlint-visualstudio/blob/master/src/Integration/Connection/UI/ConnectionInfoDialogView.xaml#L12
Perhaps asmref doesn't pick up xaml references.

Reply:
That is odd. If I remember correctly, `asmref` gets the list of references from the assembly manifest metadata, which I would expect would contain all referenced assemblies.
Maybe references that are only in compiled baml don't get included in the assembly manifest.